source/chapters/library.rst

```diff
@@ -866,6 +866,90 @@ different location. Select :guilabel:`Use Default Library` to reset.
   Mac partition mounted in Linux, you can load it and use your iTunes tracks and
   playlists as well.
 
+.. _import-serato-metadata:
+
+Importing metadata from Serato
+------------------------------
+
+.. sectionauthor::
+   Ukpai Ugochi <outreachy applicant>
+
+Serato produces DJ softwares used for digital music mixing as well as digital vinyl record
+mixing for DVS records. While Serato is neither open source or customizable, It is possible 
```

Comment:
for ease of reviewing, can you put every sentence on its own line?

Reply:
(without word wrap.  That way each line of the pull request is a single sentence)

Reply:
Am on that!